test/dlc_tests/container_tests/bin/benchmark/run_pytorch_training_performance_gpu_imagenet

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+if [ "$PYTHON_VERSION" -eq 2 ]
+then
+  exit 0
+fi
+TIMESTAMP=$(date "+%Y-%m-%d-%H-%M-%S")
+HOME_DIR=/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs/gpu
+LOG_FILE=image_results_${COMMIT_INFO}_${TIMESTAMP}.txt
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+function trigger_test() {
+  time_out=$1
+  epochs=0
+  if [ "$2" == "run" ]
```

Comment:
Please make the condition for the other type: "prep" more explicit by checking if [ "$2" == "prep" ]